apps/storefront/src/pages/QuickOrder/components/SearchProduct.tsx

```diff
@@ -105,18 +105,18 @@ export default function SearchProduct({ addToList }: SearchProductProps) {
     setProductList([...productList]);
   };
 
-  const handleAddToListClick = async (products: CustomFieldItems[]) => {
+  const handleAddToListClick = async (product: CustomFieldItems) => {
     try {
       setIsLoading(true);
-      await calculateProductListPrice(products);
-      await addToList(products);
+      await calculateProductListPrice([product]);
+      await addToList(product);
     } finally {
       setIsLoading(false);
     }
   };
 
-  const handleProductListAddToList = async (products: CustomFieldItems[]) => {
-    await handleAddToListClick(products);
+  const handleProductListAddToList = async (product: CustomFieldItems) => {
+    await handleAddToListClick(product);
   };
```

Comment:
do we need this indirection? can't we just pass `handleAddToListClick` directly?